lusid-sdk-python/tests/test_transactions.py

```diff
@@ -0,0 +1,224 @@
+import unittest
+import requests
+import json
+import uuid
+import os
+from datetime import datetime
+import pytz
+import lusid
+import lusid.models as models
+from unittest import TestCase
+from msrest.authentication import BasicTokenAuthentication
+from InstrumentLoader import InstrumentLoader
+
+
+try:
+    # Python 3.x
+    from urllib.request import pathname2url
+except ImportError:
+    # Python 2.7
+    from urllib import pathname2url
+
+
+class TestTransactionsAPI(TestCase):
```

Comment:
Rename to something more in line with file name and use this instead of self for all class variables

Reply:
done the rename, but as discussed we kept self in place to keep code tidy